source/ranecu.f90

```diff
@@ -73,6 +73,38 @@ subroutine ranecu(rvec,len,mcut)
 
 end subroutine ranecu
 
+! Init the random generator, that is, set seeds with proper value checks as described in [2]
```

Comment:
Maybe mention in the comment here that if not given, then the 2nd seed is calculated automatically.

Reply:
It is generally bad practice to make comments explaining the obvious, but I suppose. I think it is more important to put it into a developer manual (or a usage description at the top of the module like I've done for `hdf5_output` and `strings`.

Reply:
Added a usage comment at the top.